kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/MessageFetchContext.java

```diff
@@ -97,13 +97,16 @@ protected MessageFetchContext newObject(Handle<MessageFetchContext> handle) {
     private AtomicBoolean hasComplete;
     private AtomicLong bytesReadable;
     private DelayedOperationPurgatory<DelayedOperation> fetchPurgatory;
+    private String namespacePrefix;
```

Comment:
We should add `namespacePrefix = null` in `recycle()` method.

Reply:
done